compat/js/siteorigin-panels-layout-block.js

```diff
@@ -275,15 +278,22 @@ registerBlockType('siteorigin-panels/layout-block', {
     var onLayoutBlockContentChange = function onLayoutBlockContentChange(newPanelsData) {
       if (!_.isEmpty(newPanelsData.widgets)) {
         // Send panelsData to server for sanitization.
-        jQuery.post(soPanelsBlockEditorAdmin.sanitizeUrl, {
-          action: 'so_panels_layout_block_sanitize',
-          panelsData: JSON.stringify(newPanelsData)
-        }, function (sanitizedPanelsData) {
-          if (sanitizedPanelsData !== '') {
-            setAttributes({
-              panelsData: sanitizedPanelsData
-            });
+        jQuery.post(panelsOptions.ajaxurl, {
```

Comment:
@gregpriday, you sent me a message about this the other day and mentioned this had changed and wasn't sure if it was correct or not. Was it correctly changed? sanitizeUrl still exists [here](https://github.com/siteorigin/siteorigin-panels/blob/feature/improved-yoast-compat/compat/layout-block.php#L59-L60).

Reply:
I just had a look and `ajaxurl` does have a nonce added, so this should be fine. I imagine @braamgenis changed this to be able to hook into the action that also gives the rendered version of panels_data.